Change Tristan to Morgan, Update Taylor's picture

kelnera opened the pull request:
@osuosl/web-reviewers Just a couple of staff page updates. I've updated @fahlmant 's picture and @LyonesGamer 's profile. Also, Morgan I don't think you're subscribed to the cass-pelican, but I have a pr open there that updates your CASS profile. If you want to glance through them both and make sure I didn't miss anything, that would be great.

LyonesGamer commented:
:+1:

ramereth commented on content/about/about_people.rst:
```diff
@@ -162,7 +162,7 @@ pwnguin
 ------------------
 
-.. image:: /images/tfahlman.jpg
+.. image:: /images/tfahlman2.jpg
```
Can you please just replace the current file with the new one?

kelnera commented:
@osuosl/web-reviewers I've now updated the remaining bio photos and resolved merge conflicts. @ramereth I also changed Taylor's photo's name. Let me know if there's anything else.
